src/constants.js

```diff
@@ -38,6 +38,8 @@ export const CALL = {
 		AUDIO_STARTING: 4,
 		FAILED: 5,
 	},
+
+	REACTIONS: ['❤️', '🎉', '👏', '👍', '👎', '😂', '🤩', '🤔', '😮', '😥'],
```

Comment:
```suggestion
	REACTIONS: ['❤️', '🎉', '👏', '👍', '😂', '🤩', '🤔', '😮', '😥'],
```
Not nice

Reply:
As per design call we want a "not nice" option.

Reply:
How about 😐 instead of 👎

Reply:
I'd avoid that at all and require people who want to express dissent to do it:
* not while someone is presenting speaking
* in a more nuanced way than just an emoji reaction

Reply:
Manager says: "User x quit"

You want to immediately react to that visually.

I agree that we shouldn't have the 💩 emoji, but a thumbs down also for getting feedback like "should we do or not" is not to intrusive/negative as we agreed in the design planning call.

Reply:
"Guns don't kill people, people do"

I agree that "thumb down" is the most neutral emoji we could offer here for negative reactions. It's up to users to decide, whether they're going to use it or not.

If we'll dig deeper, other emojis could be also considered as inappropriate:
https://mashable.com/article/emoji-culture-world

Reply:
So neither Teams nor Jitsi have a thumbs down reaction like this. I agree with @marcoambrosini also that a simple thumbs down reaction is not very productive without elaboration. On GitHub e.g. it’s often very frustrating to see thumbs down without further explanation.

We have the 🤔 emoji which is enough, and if in doubt I would rather go with what other alternatives use.

Reply:
![image](https://user-images.githubusercontent.com/213943/233080910-2cf11e7b-5535-4561-9b7d-573d4fcbcf5a.png)

Reply:
The code point is moved to the backend now (so it can be consumed by clients in similar fashion and be toggled based on the server support) at https://github.com/nextcloud/spreed/pull/9337/files#diff-d8666c6d9eaabdff763631d59b9df820f9625645a705199e82ca1844afe0eeb4R130

However, I'm strongly against censoring into a positive only reaction set not allowing to express disconsent or gather a quick yay/nay overview.